spec/models/section_spec.rb

```diff
@@ -41,4 +41,100 @@
       end
     end
   end
+
+  context 'when sections are created and destroyed' do
+    before do
+      allow(EventSender).to receive(:send_event)
+      @section = create(:section)

```

Comment:
These should be in the test cases, not in the setup. ie the section_added test case should create a section, and the section_removed test case should destroy one.